deactivate.php

```diff
@@ -47,12 +52,17 @@ function() {
 				$value = maybe_unserialize( $row->option_value );
 				if ( is_array( $value ) ) {
 					$value_flipped = array_flip( $value );
-					unset( $value_flipped[ str_replace( WP_PLUGIN_DIR . '/', '', SQLITE_MAIN_FILE ) ] );
+					$items         = array_reverse( explode( DIRECTORY_SEPARATOR, SQLITE_MAIN_FILE ) );
+					$item          = $items[1] . DIRECTORY_SEPARATOR . $items[0];
+					unset( $value_flipped[ $item ] );
```

Comment:
Why this change? 🤔

Reply:
This works correctly in environments where plugins are symlinked into `wp-content/plugins`. (And, of course, in environments where they're simply installed there.)

Reply:
Ah OK! Makes sense now. Thank you for the insights! 👍